controllers/noobaa-rules.yaml

```diff
@@ -1,76 +1,70 @@
          labels:
            origin: data-federation-service
-         record: noobaa_num_buckets_claims
+         record: namespace:noobaa_buckets_claims:max
        - expr: |
-           NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_bucket_claims
+         record: namespace:noobaa_unhealthy_bucket_claims:max
        - expr: |
-           NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: job:noobaa_total_unhealthy_namespace_buckets
+         record: namespace:noobaa_unhealthy_namespace_buckets:max
        - expr: |
-           NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_accounts_num{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: job:noobaa_namespace_bucket_count
+         record: namespace:noobaa_accounts:max
        - expr: |
-           NooBaa_accounts_num{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_total_usage{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_accounts_num
+         record: namespace:noobaa_usage:max
        - expr: |
-           NooBaa_total_usage{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_total_usage
+         record: namespace:noobaa_unhealthy_namespace_resources:max
        - expr: |
-            NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_namespace_resources{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_namespace_resources
+         record: namespace:noobaa_namespace_resources:max
        - expr: |
-           NooBaa_num_namespace_resources{namespace="redhat-data-federation"}
+           max(NooBaa_num_namespace_buckets{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_namespace_resources
+         record: namespace:noobaa_namespace_buckets:max
        - expr: |
-           NooBaa_namespace_resource_status{namespace="redhat-data-federation"}
+           count(NooBaa_namespace_bucket_status{namespace="redhat-data-federation"} == 0)
```

Comment:
how is it different from `namespace:noobaa_unhealthy_namespace_buckets:max` at L17-21?

Reply:
+1, needs to be removed. We already have count.

Reply:
I missed this somehow, my bad, removed.